resources/img/hud/priority-card.json

```diff
@@ -3,16 +3,21 @@
     [
         {
             "type": "economy",
-            "texte": [
+            "path": "/img/hud/priority-card-economy.png",
+            "text": [
+                "ECONOMY",
```

Comment:
I don't see why we need this at the beginning of each categorie, we already have that in the type. We can grab it by doing data["type"] or something similar

Reply:
"ECONOMY" at the beginning is for the title that will be print on the priority card, it's a text that could change in the future (I am not sure we will keep this one) , if we remove it and use data["type"] we won't be able to change it.

Reply:
I think this might be in another field (such as name, and renaming text by filesNames). I let you resolve that if you think it's ok like that